src/Build/Logging/FancyLogger/FancyLogger.cs

```diff
@@ -145,6 +147,13 @@ void eventSource_ErrorRaised(object sender, BuildErrorEventArgs e)
             node.Log();
         }
 
+        void console_CancelKeyPressed(object? sender, ConsoleCancelEventArgs eventArgs)
+        {
+            // Clear screen
+            FancyLoggerBuffer.Terminate();
```

Comment:
Could we just replace all this with shutdown? Would be nice to have some summary information.